amqp-0-9-1/doc/p_dev_java_client_amqp.md

```diff
@@ -653,140 +651,6 @@ To Use the Java AMQP Client Library
     }
     ```
 
-Migrate Java AMQP Applications to KAAZING Gateway 5.0
```

Comment:
@sanjay-saxena - please verify that removing this 3.x to 4.x migration info is okay. Basically, we made the package name changes in 4.x so it doesn't apply to migrating 4.x to 5.0.x as 5.0.x has identical package names.